starport/templates/add/new.go

```diff
@@ -26,7 +26,7 @@ func New(opts *Options) (*genny.Generator, error) {
 	return g, nil
 }
 
-const placeholder = "// this line is used by starport scaffolding"
+const placeholder = "// this line is used by starport scaffolding # 1"
```

Comment:
this was needed otherwise finding `placeholder` in the source code actually may end up with finding `placeholderN`